src/sentry/models/identity.py

```diff
@@ -36,11 +36,43 @@ class IdentityProvider(Model):
     """
 
     __include_in_export__ = False
+    organization = FlexibleForeignKey("sentry.Organization", blank=True, null=True)
 
+    # To be renamed to "provider" because this is a bad column name
     type = models.CharField(max_length=64)
+    # Replacing "type"
+    provider = models.CharField(max_length=64, null=True)
```

Comment:
Do you have an argument for renaming this? I don't think we actually need to rename this? If we _are_ going to rename this, I would prefer to see this done outside of this PR.

Also I actually think this name is fine no? It is the type of identity provider, as described in the doc comment above

> The IdentityProvider is unique on the type of provider (eg github, slack, google, etc).

Reply:
What's the different between this two sentences?

> The IdentityProvider is unique on the type of provider (eg github, slack, google, etc).

vs

> The IdentityProvider is unique for each provider (eg github, slack, google, etc).

What does the word "type" means, or add to the first sentence?

Reply:
It adds specificity and avoids stutter. Imagine you did this

```py
provider = IdentityProvider.objects.get(id=1)

# Using provider as the provider identifier, has stutter
print(provider.provider)

# Using type as the provider identifier, no stutter is more clear
print(provider.type)
```